jake/ossindex/ossindex.py

```diff
@@ -98,11 +98,15 @@ def call_ossindex(self, purls: Coordinates):
         response = requests.post(self.get_url(), data=json.dumps(
             data), headers=self.get_headers())
       else:
-        (username, password, _) = config_file.get_config_from_file()
+        auth = config_file.get_config_from_file(
+            {"Username",
+             "Password"},
+            ".oss-index-config")
+
         response = requests.post(self.get_url(),
                                  data=json.dumps(data),
                                  headers=self.get_headers(),
-                                 auth=(username, password))
+                                 auth=(auth["Username"], auth["Password"]))
       if response.status_code == 200:
         self._log.debug(response.headers)
         first_results = json.loads(response.text, cls=ResultsDecoder)
```

Comment:
trivial: Unsure how to comment on line of unchanged code, but we might want to add a space on line 152 to `" expired" so that it reads as `updated in cache because TTL expired`:
![image](https://user-images.githubusercontent.com/10136383/70361673-3930f880-1837-11ea-933f-501f72b5cb2b.png)

Reply:
In: 8946a68